### PR TITLE
Added support for any given SPL, notably wum.bo

### PR DESCRIPTION
**Pull request by staccDOTsol**
This PR allows holaplex to:

1. collect quote ticker and quote mint from user
2. can also later edit
3. that's used all over the place to both list and buy/sell in that given mint

it's built to allow strata foundation integrations, but applies to anything fungible :)

Anywho, outstanding ittems:

1. improve styling
2. prices still are $SOL. The @strata-foundation/react npm package does all this but can't find code. there's likely code in dox. 
3. idk mewn

Cheers

### CODE REVIEW

**Comment by staccDOTsol** [comment]
Hi maintainers!

I  know it was ballsy to replace the action on holaplex.com frontpage, but the reason is twofold:

Your leadmaintainer mentioned some promo and asked me to include auctions in the PRs,

but more importantly the new meta breaks the feature cards for existing stores. My og commits had removed the features entirely I think, whereas now I'm just defaulting to two stores. 

:)

**Comment by dandlezzz** [comment]
@staccDOTsol we handle promos with our CMS, no need to update the code.  Maybe that wasn't clear. Please remove  any changes to the featured stores component.

File: pages/storefront/edit.tsx
Comment (dandlezzz) [inline pages/storefront/edit.tsx:292]: Hmm, should we let people change mints on a store? Might be better to have them make a new store, @kristianeboe thoughts?

File: pages/storefront/new.tsx
Comment (dandlezzz) [inline pages/storefront/new.tsx:268]: Let's incorporate the SPL-token registry here. Give people a logo and official mint hash so they can confirm.

File: src/common/components/SolBalance.tsx
Comment (dandlezzz) [inline src/common/components/SolBalance.tsx:17]: Let's leave this sort of change for another Pull Request. Balance can be in SOL for now.

File: src/common/components/elements/ActivityContent.tsx
Comment (dandlezzz) [inline src/common/components/elements/ActivityContent.tsx:136]: This upgrade is blocked by https://github.com/holaplex/solana-indexer/pull/171, once merged. We can show the mint of the token that the bid was in.

**Review by dandlezzz — CHANGES_REQUESTED** [review]
Amazing work. Little more touchup to get it production ready. But this is awesome!

File: pages/storefront/edit.tsx
Comment (kristianeboe) [reply pages/storefront/edit.tsx:292]: Good question. I think only new stores should be allowed to set SPL mint. However they might need to upload a custom icon akin to the SOL circles, and those should be able to be updated from the edit page.

After launching this to production I suggest we at Holaplex offer a grace period where we offer to help already existing stores to change their mints once.

**Comment by staccDOTsol** [comment]
> @staccDOTsol we handle promos with our CMS, no need to update the code. Maybe that wasn't clear. Please remove any changes to the featured stores component.

hi, am reviiewing all this now.

Yeah, I didn't og change the behavior - what I did was remove the features entirely, cuz the new meta breaks the auction lookup. Y'all need to run a migration to retro apply default WSOL

File: pages/index.tsx
Comment (staccDOTsol) [reply pages/index.tsx]: Done - again review other comment, more work here before it'll load the index with meta updates. defer to you guys I don't know the database stuff

File: pages/storefront/edit.tsx
Comment (staccDOTsol) [reply pages/storefront/edit.tsx:292]: I guess the complication here is in the lookup for naming and pricing? 

Or is there business logic behind restricting an edit?

File: pages/storefront/new.tsx
Comment (staccDOTsol) [reply pages/storefront/new.tsx]: agree, @dandlezzz can you link your code relevant again

Comment (staccDOTsol) [reply pages/storefront/new.tsx]: I'm also unsure re: the wum.bo stuff - remember I og wrote this on noah's request, so I'd like to additionally write in their stuff, but it's pretty easy once I solve for dependency hell

File: src/common/components/SolBalance.tsx
Comment (staccDOTsol) [reply src/common/components/SolBalance.tsx:17]: Deal, was rushing to prod ha

File: src/common/components/elements/ActivityContent.tsx
Comment (staccDOTsol) [reply src/common/components/elements/ActivityContent.tsx:136]: and/or wum.bo ;)

Comment (staccDOTsol) [reply src/common/components/elements/ActivityContent.tsx:136]: in wumbo's case it's onchain afaik as meta

File: src/modules/theme/index.ts
Comment (staccDOTsol) [reply src/modules/theme/index.ts:121]: oops, ctrl+shift+h maybe

Comment (staccDOTsol) [reply src/modules/theme/index.ts:121]: literally removing " sol" hahah

**Comment by dandlezzz** [comment]
@staccDOTsol can you please remove the unchanged files.

**Comment by staccDOTsol** [comment]
notably your deliverables don't require us to ship both strata and coiingeckp pricing, it's woth considering removing their ui  stuff for now so we can move forward.

**Comment by staccDOTsol** [comment]
https://github.com/holaplex/metaplex/pull/346#issuecomment-1061433644

**Comment by staccDOTsol** [comment]
https://twitter.com/devinity0/status/1501073956341698560?s=20&t=60gYgFV50HYBrLodk6Fb6g

**Pull request closed without merging**
